docs/products/clickhouse/reference/supported-interfaces-drivers.rst

```diff
@@ -0,0 +1,33 @@
+Interfaces and drivers supported in Aiven for ClickHouse®
+=====================================================================
+
+Find out what technologies and tools you can use to interact with Aiven for ClickHouse®.
+
+.. _clickhouse-interfaces:
+
+Interfaces (protocols)
+----------------------
+
+Aiven for ClickHouse® supports the following fundamental underlying interfaces (protocols):
```

Comment:
Maybe we could add something here about how Aiven does not support connecting to ClickHouse without TLS (ie plaintext) for security reasons (this is a deliberate choice to protect the customer from doing something dangerous and unsecure).

Reply:
And also we do not support gRPC right now, might be worth explicitly saying this.  I believe it is good to be more upfront about unsupported Clickhouse things.